pkg/skaffold/util/port.go

```diff
@@ -32,7 +32,6 @@ import (
 // See https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt,
 func GetAvailablePort(port int, forwardedPorts *sync.Map) int {
 	if isPortAvailable(port, forwardedPorts) {
-		forwardedPorts.Store(port, true)
```

Comment:
These stores are necessary as there are race conditions: port-forwarding gets an available port but doesn't use it immediately